lib/analyze.js

```diff
@@ -20,7 +20,8 @@ commonWordsArray.forEach(w => {
   commonWords[w] = 1
 })
 
-module.exports = function (url, body, returnSource) {
+module.exports = function (data) {
```

Comment:
I'm reflexively opposed to naming a variable `data` because it tells the reader absolutely nothing; `data` could be just about anything.

`renderedResult`, `requestResult`, or something similar would be more clear.

Reply:
I reverted to the previous function parameters. 
I was trying to be too clever.